src/calendar/header/index.js

```diff
@@ -42,11 +42,17 @@ class CalendarHeader extends Component {
     if (!this.props.hideArrows) {
       leftArrow = (
         <TouchableOpacity onPress={this.substractMonth} style={this.style.arrow}>
-          <Image source={require('../img/previous.png')} style={this.style.arrowImage}/></TouchableOpacity>
+          {this.props.renderArrow
+            ? React.cloneElement(this.props.renderArrow('left'))
```

Comment:
What's the intention behind cloning the element here if you're not adding properties to it? Versus just
```js
? this.props.renderArrow('left')
```